website/src/data/users.js

```diff
@@ -143,6 +143,12 @@ const users = [
     infoLink: 'https://medium.com/asurion-product-development',
     pinned: true
   },
+  {
+    caption: 'BlueDot',
+    image: 'https://www.bluedotcorp.com/wp-content/uploads/2021/03/logo-blue-2.svg',
+    infoLink: 'https://www.bluedotcorp.com/,
```

Comment:
The Link is not properly in quotations.
```suggestion
    infoLink: 'https://www.bluedotcorp.com/',
```